src/wp-admin/options-general.php

```diff
@@ -139,6 +139,7 @@
 			class="<?php echo esc_attr( $classes_for_button ); ?>"
 			data-alt-classes="<?php echo esc_attr( $classes_for_button_on_change ); ?>"
 			data-size="512"
+			data-choose="<?php esc_attr_e( 'Choose a Site Icon' ); ?>"
 			data-choose-text="<?php esc_attr_e( 'Choose a Site Icon' ); ?>"
```

Comment:
Why not reusing the existing 'data-choose-text=' instead of adding the same string in 'data-choose'?